tests/synthesizer/conftest.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from aioresponses import aioresponses, CallbackResult
```

Comment:
we'll need to add `aioresponses` as a dev dependency:

`poetry add aioresponses --group dev`